cli/cluster/yaml.go

```diff
@@ -29,16 +33,85 @@ func (collector YamlCollector) Collect() (*Config, error) {
 	return config, nil
 }
 
+// yamlCollectorDecorator is a wrapper over integrity.DataCollector
+// implementing Collector.
+type yamlCollectorDecorator struct {
+	rawCollector integrity.DataCollector
+}
+
+// Collect collects configuration from raw data interpretening it
+// as YAML.
+func (collector yamlCollectorDecorator) Collect() (*Config, error) {
```

Comment:
This module will probably become a separate library.

Let's make it public and add tests for it.

Reply:
Done